applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -133,6 +150,23 @@ put(Context) ->
 delete(Context, _RateId) ->
     crossbar_doc:delete(Context).
 
+-spec validate_number(ne_binary(), cb_context:context()) -> cb_context:context().
+validate_number(Phonenumber, Context) ->
+    case length(re:replace(Phonenumber, "[^0-9]", "", [global, {return, list}])) >= 7 of 
```

Comment:
OK, got you. 
Will go your way.
And will make pull request to remove US country code from default reconcile_regex since Kazoo spreads around the globe with speed of the light. And we'll see what you guys will tell about that :)